common/ast/src/parser/statement.rs

```diff
@@ -741,6 +741,24 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
         },
     );
 
+    // share statements
+    let create_share = map(
+        rule! {
+            CREATE ~ SHARE ~ (IF ~ NOT ~ EXISTS )? ~ #peroid_separated_idents_1_to_2 ~ ( COMMENT ~ "=" ~ #literal_string)?
+        },
+        |(_, _, opt_if_not_exists, (catalog, share), comment_opt)| {
```

Comment:
Do we need to include the catalog in `share`? I thought we could handle them in the same way as `stage`.

Reply:
> Do we need to include the catalog in `share`? I thought we could handle them in the same way as `stage`.

@leiysky @dantengsky

Reply:
Got it:)